src/components/forms/AppImageUpload.vue

```diff
@@ -2,7 +2,9 @@
   <div>
     <AppLabel v-if="label" :label="label" :required="required" />
     <div class="flex items-start gap-4">
-      <div class="flex-0 basis-28 bg-grey-light">
+      <div
+        class="flex-0 basis-28 rounded border border-primary-40 bg-primary-20"
```

Comment:
It's really small but if you look closely you can see that the image is overlapping the rounded border. You want to add either `overflow-hidden` here or `rounded` to the `<img>` so it respects the rounding, the former is probably preferable in this instance

Reply:
Yes! Added `overflow-hidden`, thanks! I saw the tiny overlapping edge and forgot to fix it.